packages/seacas/applications/blot/cgi/vdicps.f

```diff
@@ -3428,43 +3430,44 @@ SUBROUTINE PSTA2C(ASCI,CHARAC)
       return
       end
       SUBROUTINE PSTI2C(INT,NDIGIT,ISTR)
-C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C
-C
-C PSTI2C           - convert positive integer to decimal character
-C                    string equivalent
-C
-C ENVIRONMENT      - COMPUTER-INdependent
-C
-C ENTRY CONDITIONS - int = positive integer to be converted
-C                  ndigit = number of digits to be produced in string
-C                           form (pad left with zeros)
-C                  istr = character string of at least ndigit characters
-C
-C CALLS            -
-C
-C EXIT CONDITIONS  - istr contains decimal-string equivalent of int
-C                       (ndigits left-justified in istr)
-C
-C NARRATIVE        - This routine modified 10/89  S.L.Thompson
-C
-C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C
+C     C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C C
+C     
+C     PSTI2C           - convert positive integer to decimal character
+C     string equivalent
+C     
+C     ENVIRONMENT      - COMPUTER-INdependent
+C     
+C     ENTRY CONDITIONS - int = positive integer to be converted
+C     ndigit = number of digits to be produced in string
+C     form (pad left with zeros)
+C     istr = character string of at least ndigit characters
+C     
+C     CALLS            -
+C     
+C     EXIT CONDITIONS  - istr contains decimal-string equivalent of int
+C     (ndigits left-justified in istr)
+C     
+C     NARRATIVE        - This routine modified 10/89  S.L.Thompson
```

Comment:
Do you need to update this comment?

Reply:
Probably need to take out all of the "modified ..." type comments at some point.  Is neat to see the names of some of the old-timers who worked on some of these routines in the past...

Reply:
@gsjaardema I don't mind seeing the history :-)